etc/ctrl.with.edge.yml

```diff
@@ -69,11 +69,3 @@ edge:
       # (optional, defaults to 5) The length of time that a Ziti Edge Router enrollment should remain valid. After
       # this duration, the enrollment will expire and not longer be usable.
       durationMinutes: 5
-  persistence:

```

Comment:
should this be left around for a few more cycles? seems like this could be left commented out for historical reasons/migration?

Reply:
I'll add it back, commented out for now